src/main/java/com/raeic/embarker/land/models/StakedChunkManager.java

```diff
@@ -80,6 +67,7 @@ public StakedChunk findOneIgnoreCache(int coordX, int coordZ, String worldName)
                     Timestamp firstStaked = results.getTimestamp("firstStaked");
                     Timestamp lastUpdated = results.getTimestamp("lastUpdated");
                     result = Globals.stakedChunks.create(coordX, coordZ, worldName, ownerUniqueId, firstStaked, lastUpdated);
+                    result.save();
```

Comment:
I think writing to DB as the only way to insert into the cache is not great but if it works, it works \o/

Reply:
(: